src/coreclr/gc/gc.cpp

```diff
@@ -2855,12 +2861,18 @@ size_t     gc_heap::interesting_mechanism_bits_per_heap[max_gc_mechanism_bits_co
 
 mark_queue_t gc_heap::mark_queue;
 
+#ifdef USE_REGIONS
 bool gc_heap::special_sweep_p = false;
+#endif //USE_REGIONS
+
+int gc_heap::loh_pinned_queue_decay = LOH_PIN_DECAY;
 
 #endif // MULTIPLE_HEAPS
 
 /* end of per heap static initialization */
 
+const size_t uninitialized_end_gen0_region_space = (size_t)(-1);
```

Comment:
this is only used for regions, you could move it to the #ifdef USE_REGIONS right above.